src/applications/find-va-forms/containers/SearchForm.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { updateQuery } from '../actions';
+
+function SearchForm({ query, updateQuery: onQueryChange }) {
+  return (
+    <form
+      name="find-va-form"
+      className="vads-l-grid-container vads-u-padding--0"
+    >
+      <label htmlFor="va-form-query" className="vads-u-margin--0">
+        Keyword, form name, or number
+      </label>
+      <div className="vads-l-row">
+        <div style={{ flexGrow: 1 }} className="vads-u-margin-right--2">
+          <input
+            className="usa-input vads-u-max-width--100 vads-u-width--full"
+            name="va-form-query"
+            type="text"
+            value={query}
+            onChange={event => onQueryChange(event.target.value)}
+          />
+        </div>
+        <div>
```

Comment:
Do we need this extra `div` here? :)

Reply:
Oh, probably not. I'll try removing that.

Reply:
Ah, yep, looks like it is necessary for the flexbox to work -

![image](https://user-images.githubusercontent.com/1915775/69551905-7fe73e80-0f6b-11ea-8e1b-3b90ee1027f9.png)